portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/ManagementToolbar.testcase

```diff
@@ -223,43 +223,43 @@ definition {
 	@description = "LPS-147851. Assert the order button can display the correct icons"
 	@priority = "4"
 	test CanDisplayCorrectOrderIcons {
-		task ("And clicks on the order button") {
+		task ("And When order dropdown menu is opened") {
 			Click(
 				key_filter = "Order",
 				locator1 = "ClaySamplePortlet#DISPLAY_CONTEXT_FILTER_OR_ORDER",
 				value1 = "Order");
 		}
 
-		task ("And selects the descending order") {
+		task ("And When descending order is selected") {
 			Click(
 				key_item = "Descending",
 				locator1 = "ManagementBar#ACTIVE_DROPDOWN_ITEM",
 				value1 = "Descending");
 		}
 
-		task ("Then the order icon will be displayed accordingly") {
+		task ("Then the order list down icon will be displayed") {
```

Comment:
Here `order icon will be displayed accordingly` isn't a specific assertion. When we write in BDD, we want to be more specific on what we're asserting. We want to avoid catch all phrases and fluff.

Reply:
I may even go a step further and say "Then the order list down icon will be displayed in the order button". I had trouble trying to toggle the feature flag off so I wasn't able to verify the xpath would scope to within the button/link.